### PR TITLE
Use content set labels for repos

### LINKED ISSUE #142: Repo labes
Fix mismatch between repos labels seen on client vs. pulp.
Repo vs. content set.

### PR DESCRIPTION
**Pull request by MichaelMraka**
fixes #142

### CODE REVIEW

File: database/vmaas_db_postgresql.sql [hunk 1]
```diff
@@ -221,16 +221,15 @@ CREATE TABLE IF NOT EXISTS certificate (
...
-  arch_id INT NULL,
-  releasever TEXT NULL,
+  content_set_id INT NOT NULL,
+  basearch_id INT NOT NULL,
+  releasever TEXT NOT NULL,
   eol BOOLEAN NOT NULL,
   revision TIMESTAMP WITH TIME ZONE NOT NULL,
   certificate_id INT NULL,
   PRIMARY KEY (id),
+  UNIQUE (content_set_id, arch_id, releasever),
```
Comment (ggainey) [inline database/vmaas_db_postgresql.sql]: Still refers to arch_id, breaks on db-create

**Comment by ggainey** [comment]
database/repository_store.py still knows repo.label in a number of places, and I'm not comfortable I can fix it quickly-reliably - please fix and update!

**Comment by jdobes** [comment]
@ggainey ad repo.label in reposcan code - I will create second PR that will fix this and then you can merge  both PRs at once, stay tuned

**Comment by ggainey** [comment]
#158 and #165 merged, tested, and pushed together as of 14c400628430ac7ed41bd609e85792e566864205

**Pull request closed without merging**